frontend/src/components/applyForm/ApplyFormMessage.tsx

```diff
@@ -0,0 +1,55 @@
+import { useTranslations } from "next-intl";
+import { Alert } from "@trussworks/react-uswds";
+
+import { FormValidationWarning } from "./types";
+
+export const ApplyFormMessage = ({
+  error,
+  validationWarnings,
+  submitted,
+}: {
+  error: boolean;
+  validationWarnings: FormValidationWarning[] | null;
+  submitted: boolean;
+}) => {
+  const t = useTranslations("Application.applyForm");
+  const errorMessage = t.rich("errorMessage", {
+    "email-link": (content) => (
+      <a href="mailto:simpler@grants.gov">{content}</a>
+    ),
+    p: (content) => <p>{content}</p>,
+  });
+  if (!submitted) {
```

Comment:
it looks like this submitted param is set to true when the application form is saved - I imagine that's intentional. The difference between `form data submission` and `application form submission` is potentially confusing. In this case since the messaging seems to be save specific, I wonder if `saved` would be a more accurate and less confusing name? Or is the idea to extend this to encompass saving and submitting application forms?

Reply:
I used `submitted` b/c it isn't guaranteed to save, but given the context of the app submission, agree that it could be confusing. Updated to `saved`, thanks.